pkg/controller/seed-controller-manager/monitoring/resources.go

```diff
@@ -49,7 +49,7 @@ func (r *Reconciler) getClusterTemplateData(ctx context.Context, client ctrlrunt
 	}
 
 	// Konnectivity is enabled if the feature gate is enabled and the cluster flag is enabled as well
```

Comment:
NIT: Since you removed the feature gate, this comment is invalid :)

Reply:
thx, fixed ;)